source/jsonnet/northern-ireland/individual/blocks/school/study_location_type.jsonnet

```diff
@@ -72,6 +72,18 @@ local proxyDescriptionStudy = {
         ],
       },
     },
+    {
+      goto: {
+        block: 'travel-to-study-location',
+        when: [
+          {
+            id: 'study-location-type-answer',
+            condition: 'equals',
+            value: 'No fixed place',
+          },
+        ],
+      },
+    },
```

Comment:
This isn't needed if we update the existing rule to:
```js
    {
      goto: {
        section: 'End',
        when: [
          {
            id: 'study-location-type-answer',
            condition: 'equals',
            value: 'At or from home',
          },
        ],
      },
    },
```
This is because the default routing rule of `block: 'travel-to-study-location'` handles both unanswered and `No fixed place`, as there is only 3 answer options.